src/index.html

```diff
@@ -1,17 +1,171 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport" content="width=device-width, user-scalable=no, 
+    initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+    >
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <label class="input">
+        Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            autocomplete="off" 
+            required
+          >
+        </label>
```

Comment:
checklist p2, fix in all cases
```suggestion
        <label class="input">
          Surname:
          <input 
            type="text" 
            name="surname" 
            autocomplete="off" 
            required
          >
        </label>
```